cloudpak/stable/ibm-spectrum-scale-csi-operator-bundle/case/ibm-spectrum-scale-csi-operator/inventory/ibmCSIScaleOperator/resources.yaml

```diff
@@ -149,7 +149,7 @@ resources:
     - metadata:
         name: livenessprobe
       image: cp/spectrum/scale/csi/livenessprobe
-      tag: v2.10.0
+      tag: v2.11.0
```

Comment:
liveness probe version should be still 2.10.0

Reply:
Done